storage/database.go

```diff
@@ -201,6 +211,8 @@ func (d *db) Open() error {
 	// All goroutines must be accounted for with dbOngoingTasks to receive done signal
 	go d.reportLoop()
 	go d.ongoingTick()
+	go d.repairer.Start()
```

Comment:
Maybe add an endline before calling this and explicitly mention that `Stop()` is explicitly called on close so that's why `dbOngoingTasks` is not accounting for this process?